docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Should it be: "account information, the <code>Service()</code> object" ?</p><p> </p><p>Also, the qss version breaks this up into three bullet points:</p><p> </p><ul><li><code>backends()</code>: Retrieves a list of available backends</li><li><code>get_backend("&lt;backend_name&gt;")</code>: Select your target backend, where&nbsp;<code>&lt;backend_name&gt;</code>&nbsp;is the name of the desired backend</li><li><code>get_backend("&lt;backend_name&gt;").target_info()</code>: Retrieve information on your selected backend, such as number of qubits, native gate set</li></ul><p> </p><p>which I think would also be helpful here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Sounds good - what is the equivalent of <code>get_backend()</code> in Cirq / <code>css</code> ?  </p>

Reply:
<p>My bad, I forgot that with <code>cirq-superstaq</code> you just do <code>cirq._superstaq.Service().target_info("ibmq_qasm_simulator")</code> for example.   </p>